assignment-section-05/classification_model/config.yml

```diff
@@ -0,0 +1,51 @@
+# Package Overview
+package_name: regression_model
```

Comment:
classification_model? or titanic_model?

Reply:
Because we called the other model `regression_model` I felt this was more consistent.

Reply:
since this is a classification task, I would anticipate that most students would change the name (regression is normally used for continuous targets). But I guess it is not so important. If it is a small tweak, change it. Otherwise leave it.